chain/network/src/routing/routing.rs

```diff
@@ -194,8 +193,9 @@ impl RoutingTableView {
 
     pub fn remove_edges(&mut self, edges: &Vec<Edge>) {
         for edge in edges.iter() {
-            assert!(edge.key().0 == self.my_peer_id || edge.key().1 == self.my_peer_id);
-            self.local_edges_info.remove(edge.key());
+            if let Some(other_peer) = edge.other(&self.my_peer_id) {
+                self.local_edges_info.remove(other_peer);
```

Comment:
now we're going to silently ignore the edges that don't start from the current peer (before we were asserting).

if this was intended - please add the comment to the function

Reply:
or maybe rename to something like 'remove_edges_if_local' ?

Reply:
Ok, I renamed the method to `remove_local_edges` and I added panic!.